exercises/phone-number/phone_number_test.go

```diff
@@ -20,6 +22,9 @@ var numberTests = []testCase{
 }
 
 func TestNumber(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, wanted %v.", testVersion, targetTestVersion)
+	}
 	for _, test := range numberTests {
 		actual, actualErr := Number(test.input)
 		if actual != test.expected {
```

Comment:
could you make it so that this is only checked if `actualErr == nil`? This would cover the "Ignore the normal return value when an error expected" point of #302